wasm/build.sh

```
@@ -0,0 +1,496 @@
+
+    # Check if we need to bundle f2c runtime
+    if [[ -z "$F2C_LIB" ]]; then
+        echo "  Warning: libf2c.a not found, will try system linking"
+        F2C_LINK="-lf2c"
+    else
+        echo "  Found f2c library: $F2C_LIB"
+        F2C_LINK="$F2C_LIB"
+    fi
+
+    # Main compilation
+    emcc $EMCC_FLAGS \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)